build-system/tasks/prettify.js

```diff
@@ -70,110 +71,85 @@ function prettify() {
  *
  * @param {string} file
  */
-function printErrorWithSuggestedFixes(file) {
+async function printErrorWithSuggestedFixes(file) {
   logWithoutTimestamp('\n');
   log(`Suggested fixes for ${cyan(file)}:`);
+  const options = await getOptions(file);
```

Comment:
nit/idea: assuming this performs fine on your machine, feel free to ignore for this comment.

What would be the most efficient way of collecting options? 
As-is this needs to hit the fs to check config for each file.

Reply:
We need to call `getOptions()` once per file because its return value is dynamic and based on the file extension, location, etc. From the [docs](https://prettier.io/docs/en/api.html#prettierresolveconfigfilepath--options), `resolveConfig()` caches its configs, so this shouldn't slow things down. Running this task against the entire repo takes ~25s on my laptop.

Reply:
oh great! thank you for the detailed answer